libafl_targets/src/sancov_cmp.c

```diff
@@ -125,3 +125,19 @@ void __sanitizer_cov_trace_const_cmp4(uint32_t arg1, uint32_t arg2) {
 void __sanitizer_cov_trace_const_cmp8(uint64_t arg1, uint64_t arg2) {
     __sanitizer_cov_trace_cmp8(arg1, arg2);
 }
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+
+void __sanitizer_cov_pcs_init(const uintptr_t *pcs_beg,
+                              const uintptr_t *pcs_end) {
+    // unused
+    // TODO implement
+}
+
+void __sanitizer_cov_trace_pc_indir(uintptr_t Callee) {
+    // unused
+    // TODO implement
```

Comment:
Reading through the documentation at <https://clang.llvm.org/docs/SanitizerCoverage.html#tracing-pcs-with-guards> it looks like this is pretty straight forward, we just want a portion of the coverage map for indirect branches and then map them as feedback.
Also, it looks like the function is called `__sanitizer_cov_trace_pc_indirect(void *callee)` in newer compilers (or in some versions and the docs are old?) so maybe we want to export both